pandas/tests/io/test_pytables.py

```diff
@@ -4999,6 +4999,62 @@ def test_read_nokey_empty(self):
             store.close()
             pytest.raises(ValueError, read_hdf, path)
 
+    # GH10143
```

Comment:
put this near tests for get_node

Reply:
Sorry, but I couldn't spot any test for get_node in the repo?

Reply:
moved